ReactNativeClient/lib/components/screens/note.js

```diff
@@ -184,6 +184,7 @@ class NoteScreenComponent extends BaseScreenComponent {
 		this.cameraView_onPhoto = this.cameraView_onPhoto.bind(this);
 		this.cameraView_onCancel = this.cameraView_onCancel.bind(this);
 		this.properties_onPress = this.properties_onPress.bind(this);
+		this.showOnMap_onPress = this.showOnMap_onPress.bind(this);
```

Comment:
please remove

Reply:
This line is important. I'm calling dispatch function inside another function, so it's taking this as that function reference. So either I've to bind the function here, or I think connect the actions with mapDispatchToProps should also work in connect. If you have any other way, please tell.

Reply:
Yes of course, sorry, I thought you had added then removed this function, but the function was already there. And indeed you'd need to bind it.

Reply:
Okay thanks, so do i need to change anything in this PR or I can start working on other issues.  And can you please confirm, if i can start working on issue #2721.

Reply:
Feel free to start working on any other issue. The code on this PR looks good but I just need to test it on emulator.

Reply:
Okay thanks. I'll start working on other issues.